pkg/kudoctl/cmd/get/instances.go

```diff
@@ -0,0 +1,129 @@
+package get
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/user"
+
+	"github.com/spf13/cobra"
+	"github.com/xlab/treeprint"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"path/filepath"
+
+	kudov1alpha1 "github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/dynamic"
+)
+
+var (
+	kubeConfig string
+	namespace  string
+)
+
+const (
+	defaultConfigPath = ".kube/config"
+)
+
+// NewGetInstancesCmd creates a command that lists the instances in the cluster
+func NewGetInstancesCmd() *cobra.Command {
```

Comment:
I don't quite understand why there are two commands, one in `pkg` and one in `cmd`.  Which one is actually run?  What's the purpose of the other one.  This all isn't necessary for this PR though

Reply:
I think Intellij must have done this somehow during refactoring. I definitely did not move any code around by myself... let me retry once more :)

Reply:
Think you're ok, the one under pkg/kudoctl/cmd has been there a while. We may want to move the functionality of listing instances out to it's own package and not have cobra details inside of pkg/kudoctl/cmd - but like @runyontr said, that's another PR.